recipes-bsp/acpi-tables/acpi-tables.bb

```diff
@@ -45,6 +52,24 @@ do_compile() {
 	done
 }
 
+do_install() {
+
+	install -d ${D}/kernel/firmware/acpi
+	for table in ${ACPI_TABLES}; do
+		dest_table=$(basename $table asl)
```

Comment:
`asl` -> `.asl` and so for the following lines.

Reply:
Done